suite-native/module-home/src/screens/HomeScreen/components/EmptyHomeRenderer.tsx

```diff
@@ -26,17 +27,27 @@ export const EmptyHomeRenderer = () => {
         return null;
     }
 
+    let ScreenContent;
+
     if (isUsbDeviceConnectFeatureEnabled) {
         // Crossroads should be displayed if there is no real device connected and portfolio tracker has no accounts
         // or if there is device connected, but not authorized (PIN enter cancelled).
         if (hasOnlyEmptyPortfolioTracker || !isDeviceAuthorized) {
-            return <EmptyPortfolioCrossroads />;
+            ScreenContent = EmptyPortfolioCrossroads;
         }
 
         if (!isPortfolioTrackerDevice && isDeviceAuthorized) {
```

Comment:
I'm pretty sure you wanna change this this to `else if`:
```suggestion
        else if (!isPortfolioTrackerDevice && isDeviceAuthorized) {
```

Reply:
good catch 6271670fb5